packages/edge-login-ui-rn/src/native/components/screens/PinLogInScreenComponent.js

```diff
@@ -91,14 +91,27 @@ export default class PinLogInScreenComponent extends Component<Props, State> {
     if (this.props.loginSuccess) {
       return null
     }
+    if (this.state.focusOn === 'List') {
+      return (
```

Comment:
This is a lot of copy/paste. Put the contents of the View in a function that you can call from within each condition (in or out of focus).